test/functional/rpc_net.py

```diff
@@ -262,6 +263,12 @@ def test_addpeeraddress(self):
         assert_equal(node.addpeeraddress(address="1.2.3.4", port=8333), {"success": False})
         assert_equal(len(node.getnodeaddresses(count=0)), 1)
 
+        self.log.debug("Test adding a second address (with a different /16) to the tried table")
```

Comment:
could be nice to pull out the `(with a different /16)` from the log & add a comment with the reasoning of why (so it maps to a different bucket) as described in https://github.com/bitcoin/bitcoin/pull/22831#discussion_r708352287

Reply:
Oops, meant to post it here. SGTM, what do you think: https://github.com/bitcoin/bitcoin/pull/22831#discussion_r709171663

Reply:
done!